tests/rules/no-subclass.ts

```diff
@@ -0,0 +1,234 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { stripIndent } from "common-tags";
+import rule = require("../../source/rules/no-subclass");
+import { ruleTester } from "../utils";
+
+ruleTester({ types: true }).run("no-subclass", rule, {
+  invalid: [
+    {
+      code: stripIndent`
+        import { AsyncSubject, BehaviorSubject, Observable, ReplaySubject, Subject, Subscriber } from "rxjs";
+        import { Scheduler } from "rxjs/internal/Scheduler";
+
+        class GenericObservable<T> extends Observable<T> {}
```

Comment:
Should the generic be included in the error?
Now, it marks `Observable` as error, TSLint the error is `Observable<T>`

Reply:
I don't really mind. IMO, it's not a big deal. As long as the highlighting is relatively tightly focused - i.e. doesn't highlight an entire expression - I'm fine with it.